src/test/regress/input/multi_outer_join.source

```diff
@@ -163,14 +163,13 @@ SELECT * FROM master_apply_delete_command('DELETE FROM multi_outer_join_left');
 SELECT * FROM master_apply_delete_command('DELETE FROM multi_outer_join_right');
 
 -- reload shards with 1-1 matching
-\copy multi_outer_join_left FROM '@abs_srcdir@/data/customer-1-15.data' with delimiter '|'
+\copy multi_outer_join_left FROM '@abs_srcdir@/data/customer-11-20.data' with delimiter '|'
```

Comment:
What is the reason for this change? Is this really required?

Reply:
To make left and right tables colocated, otherwise we can't use local join.